backend-api/app/routers/system.py

```
@@ -241,28 +238,15 @@ def _matches_hw(c: str) -> bool:
-            try:
-                task.get(timeout=90)
-            except Exception as e:
-                logger.warning("rerun_encoder_tests: task.get raised: %s", e)
-
-        await asyncio.to_thread(_wait)
-        invalidate_hw_info_cache()
-        logger.info("encoder-tests/rerun: completed")
+        try:
+            _ = task.get(timeout=90)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)